### PR TITLE
update adaptive component by face test

### PR DESCRIPTION
**Pull request by mjkkirschner**
update test and test file which was found to have an issue in previous versions as well, the test was still passing but timing out - make the test more precise by checking the resulting geometry.

screen shot 2018-10-18 at 5 04 00 pm

### CODE REVIEW

**Comment by mjkkirschner** [comment]
@alfarok @smangarole PTAL

File: test/Libraries/RevitIntegrationTests/AdaptiveComponentTests.cs [hunk 1]
```diff
@@ -181,6 +182,10 @@ public void CreateAdaptiveComponentsByPointsOnFace()
                 var adapValue = GetPreviewValueAtIndex(adapID, i) as AdaptiveComponent;
                 Assert.IsNotNull(adapValue);
             }
+            var volumeSumId = "f76b55af-52f0-451e-9819-2a51ce8a1afc";
+            var volume = GetPreviewValue(volumeSumId);
+            Assert.IsTrue(Math.Abs((double)volume - 13.39) < 0.1);
```
Comment (alfarok) [inline test/Libraries/RevitIntegrationTests/AdaptiveComponentTests.cs]: Hey @mjkkirschner, can you elaborate a little on this assertion?  Also does `AdaptiveComponentsByPointsOnFace.dyn` get used for any other tests that may be influenced by the changes made to the graph?

**Comment by alfarok** [comment]
@mjkkirschner Just a couple questions, otherwise LGTM

**Comment by mjkkirschner** [comment]
@alfarok 1 .only used in this test.
2. the assertion checks that the volume of the generated adaptiveComponents is about 13 units^2. This is just a simple way of checking that valid geometry was generated by looking at the output of a single node's preview.

**Pull request merged**